### ISSUE TITLE
Get currently played track

### ISSUE THREAD

**Issue by yoshTatass** [kaaes/spotify-web-api-android#156]
Hi all, 

is there any way to do this ?

**Comment by eddiecrawford** [comment]
No, not on this version of the web-api wrapper for android @yoshTatass 
They have not added the new web-api **player** endpoints yet.
However, you can clone the source and add the endpoints yourself:
https://developer.spotify.com/web-api/console/player/

**Comment by yoshTatass** [comment]
Hi, I've added the endpoint in SpotifyService.java, created the required objects but for some reason, I'm getting a "401 Unauthorized" error. Did I do something wrong ? do I need to do something else ?

Thanks for your help.

**Comment by maracuja-juice** [comment]
#157

**Comment by maracuja-juice** [comment]
#157
